crates/symbolicator-service/src/services/sourcemap_lookup.rs

```diff
@@ -730,6 +730,28 @@ impl ArtifactFetcher {
             }
         };
 
+        let host_string = match url.host_str() {
+            None => {
+                return CachedFileEntry {
+                    uri: CachedFileUri::ScrapedFile(RemoteFileUri::new(abs_path)),
+                    entry: Err(CacheError::DownloadError("Invalid host string".to_string())),
```

Comment:
```suggestion
                    entry: Err(CacheError::DownloadError("Invalid host".to_string())),
```

Just a small nit.

Otherwise I would move this whole block below the `scraping.enabled` check, maybe even blow the `allowed_origins`. I’m also a bit surprised that `is_valid_origin` does not catch this, but I guess the default is `*` which allows everything.

Reply:
Yeah, unfortunately it's not easily possible to to allow all scraping hosts _except_ localhost.